content/en/blog/wireshark.md

```diff
@@ -188,19 +199,12 @@ is for the names "Jason" and "Lily".
 
 ### Decoding the server-streamed response
 
-<!--
-TODO: What are the steps that the user needs to follow before the user can see
-      the decoded message as shown in the screenshot? Is the text given below correct?
-
-TODO: How many Person objects are returned as a response? It looks like two
-      from the screenshot, and the second is selected, right?
--->
 Since the `Search` RPC response is server-streaming, `Person` objects can be
 returned to the client one after another.
 
 The response port, which can differ for each RPC call, is 51035 in the sample
-capture file. Select the second `Person` returned in the response stream and
```

Comment:
The 51035 is the TCP client port, that is used for sending request and receiving response from sever. So "The response port" might be changed to "The response receiving port" or others? (I'm not very sure about the grammar.)

The HTTP2/gRPC call is based on TCP connection:

The two ends of the TCP connection are 127.0.0.1:51035 (using by TCP client) and 127.0.0.1:50051 (using by TCP server).
The gRPC client sent a request from port 51035 to server port 50051 (refer to No.12 packet).
And late, server send a response message from port 50051 to port 51035 (refer to No.16 and No.18).
Currently, the packet-list pane only displays destination port (DstPort), does not show source port.

BTW, other content of this blog looks good for me.

Reply:
If we add source port column, it will like:
![display_source_port](https://user-images.githubusercontent.com/30267326/105378186-86904500-5c46-11eb-9f3d-0fac599d25e0.png)
>Note that SrcPort and DstPort columns are added manually by users, default they are not displayed (because they can be found in packet detail pane).

Reply:
Ah, right! Ok, I'll merge in these changes (since the blog post is still a draft it won't be published yet), and address final changes in a followup PR. Thanks for your review and detailed feedback!